cql3/statements/describe_statement.cc

```diff
@@ -44,6 +46,47 @@ namespace statements {
 static const sstring KS = "system";
 static const sstring CF = "describe";
 
+static int element_order(const sstring& element) {
+    using ds = raw::describe_statement;
+    if (element == ds::KEYSPACE) {
+        return 0;
+    } else if (element == ds::TYPE) {
+        return 1;
+    } else if (element == ds::FUNCTION) {
+        return 2;
+    } else if (element == ds::AGGREGATE) {
+        return 3;
+    } else if (element == ds::TABLE) {
+        return 4;
+    } else if (element == ds::INDEX) {
+        return 5;
+    } else if (element == ds::VIEW) {
+        return 6;
+    }
+    return 999;
```

Comment:
let's switch that to std::numeric_limit<int>::max() to make the constant less magical; alternatively, perhaps we need to throw an internal error here (`on_internal_error`), because we received an unexpected constant?

Reply:
Now throwing internal error there